ci/tests/harness.h

```diff
@@ -2,6 +2,9 @@
  *
  * It is included by every benchmark file and provides the time measuring,
  * as well as standardized command-line argument parsing.
+ *
+ *  Release under Public domain.
```

Comment:
s/Release/Released/

Reply:
Fixed in [f7e847d](https://github.com/capablevms/bdwgc/pull/18/commits/f7e847d2f505b14a1a91debbcb029dc3de233831)